Integrations/SplunkPy/SplunkPy.py

```diff
@@ -205,44 +205,159 @@ def request(url, message, **kwargs):
     }
 
 
-service = None
-proxy = demisto.params()['proxy']
-if proxy:
+# COMMANDS
+
+def parse_raw_command():
+    raw = demisto.args()['raw']
+    rawDict = rawToDict(raw)
+    ec = {}
+    ec['Splunk.Raw.Parsed'] = rawDict
+    demisto.results({"Type": 1, "ContentsFormat": "json", "Contents": json.dumps(rawDict), "EntryContext": ec})
+    sys.exit(0)
+
+
+def notable_event_edit_command():
+    if not proxy:
```

Comment:
why not handle_proxy()

Reply:
Because I'm not changing any logic on this PR, just changing the structure. Logic will be later, there are many such changes to make